### PR TITLE
Use timezone aware datetimes

### PR DESCRIPTION
**Pull request by busunkim96**
Follow up to #6581. 

This was a find/replace across the repo. If a timezone aware datetime is not appropriate for your sample please let me know!

### CODE REVIEW

**Review by dandhlee — APPROVED** [review]

File: blog/introduction_to_data_models_in_cloud_datastore/blog.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def create_user(ds, username, profile):
```
Comment (dandhlee) [inline blog/introduction_to_data_models_in_cloud_datastore/blog.py:43]: nit: Need header adjustment on this file:
`# Copyright 2015, Google, Inc.`
to
`# Copyright 2015 Google, LLC.`

File: blog/introduction_to_data_models_in_cloud_datastore/wiki.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def path_to_key(datastore, path):
```
Comment (dandhlee) [inline blog/introduction_to_data_models_in_cloud_datastore/wiki.py:37]: ditto:
`# Copyright 2015, Google, Inc.`
to
`# Copyright 2015 Google, LLC.`

**Review by leahecole — COMMENTED** [review]
LGTM for Composer

**Comment by leahecole** [comment]
run/idp-sql failures unrelated

**Comment by leahecole** [comment]
rebuilding 3.10 to see if the iot is a flake

**Comment by leahecole** [comment]
Confirmed 3.10 failures unrelated!

**Pull request merged**